.codegen/service.py.tmpl

```diff
@@ -275,7 +275,7 @@ class {{.Name}}API:{{if .Description}}
           {{- end}}
         {{else -}}
         json = {{template "method-do" .}}
-        return [{{.Pagination.Entity.PascalName}}.from_dict(v) for v in json{{if .Pagination.Results}}.get('{{.Pagination.Results.Name}}', []){{end}}]
+        return {{.Response.PascalName}}.from_dict(json).{{.Pagination.Results.Name}}
```

Comment:
this change will cause downstream code to break if the response does not contain `.Pagination.Results.Name` value in it, because `None` is not iterable.